bmcremedy_connector.py

```diff
@@ -402,6 +469,56 @@ def _get_url(self, action_result, incident_number):
 
         return phantom.APP_SUCCESS, url
 
+    def _bmc_int_auth_refresh(self, action_result):
+
+        self.debug_print("In bmc_int_auth_refresh method")
+
+        config = self.get_config()
+        client_id, client_secret = config.get('client_id'), config.get('client_secret')
+        oauth_token = self._state.get('oauth_token')
+
+        if not (oauth_token and oauth_token.get("refresh_token")):
+            self._reset_the_state()
+            action_result.set_status(phantom.APP_ERROR, "Unable to get refresh token. Please run Test Connectivity again")
+            return phantom.APP_ERROR, "Unable to get refresh token. Please run Test Connectivity again"
+
+        if client_id != self._state.get('client_id', ''):
+            self._reset_the_state()
+            action_result.set_status(phantom.APP_ERROR, "Client ID has been changed. Please run Test Connectivity again")
+            return phantom.APP_ERROR, "Client ID has been changed. Please run Test Connectivity again"
+
+        refresh_token = oauth_token['refresh_token']
+
+        base_url = self._base_url.replace(self._base_url[len(self._base_url) - 4:], config.get("ports").split(",")[1].strip())
+        request_url = f'{base_url}/rsso/oauth2/token'
```

Comment:
Other endpoints are stored in consts, so I think we can move this one to consts too.

Reply:
Will add message in consts.